documentation/modules/security/proc-installing-certs-per-listener.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='proc-installing-certs-per-listener-{context}']
+= Installing your own CA certificates for listeners
+
+You can install your own certificates and public keys for Kafka brokers on a per-listener basis. These certificates can be signed by an external Certificate Authority (CA), such as your company's internal CA or a public CA, rather than the cluster or clients CAs (which are internal to {ProductName}). Installing per-listener certificates is supported for two types of listeners:
```

Comment:
Thsi would be much easier to comment on if you followed the one sentence per line policy. We should mention somewhere here that the certificates should be either wildcard certificates with appropriatte subject or have SANs matching all possibl broker names. We should possibly include some examples of how the addresses would look like so that the user knows what should be used.

Reply:
OK. I'll use one sentence per line as much as possible.

@stanlyDoge - Can you provide some example addresses in the correct format?

Reply:
I've added a new module named "SAN certificates for Kafka listeners" containing the example addresses.